### PR TITLE
Fixes #13928: Render changelogs in Rudder doc

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/13928

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ALL_VERSIONS = 2.3 2.4 2.5 2.6 2.7 2.8 2.9 2.10 2.11 3.0 3.1 3.2 4.0 4.1 4.2 4.3 5.0
```
Comment (peckpeck) [inline Makefile]: 5.1 is not included ?
Isn't there a way to detect this ? if no, you must update the new version procedure

**Comment by amousset** [comment]
Commit modified

**Pull request merged**